kubernetes/coredns.yaml.sed

```diff
@@ -54,7 +54,9 @@ data:
   Corefile: |
     .:53 {
         errors
-        health
+        health {
+          lameduck 10s
```

Comment:
wondering if we should make lameduck just a tad longer than the readiness polling interval.  e.g. 11s or 12s...